packages/Checkbox/src/Checkbox.js

```diff
@@ -13,24 +13,33 @@ const checkboxStates = {
 };
 
 const propTypes = {
+  /** Used for aria-describedby on the checkbox input  */
+  ariaDescribedBy: PropTypes.string,
```

Comment:
adding this new `ariaDescribedBy` prop in both Checkbox and Radio components as these are both typically used in a fieldset as a children of the formElement component. We can then directly apply the `ariaDescribedBy` prop to them, which is then applied to their internal `input` components.